pyspecProcScripts/integrate_limits.py

```diff
@@ -47,23 +47,27 @@ def integrate_limits(s, axis="t2",
         order not to see any diagnostic plots
 
     """
+    s.ift('t2') # move ift into function to eventually discard
+    #assert s.get_ft_prop(axis), "data must be in the frequency domain along %s!!"%axis
```

Comment:
the goal here is to get rid of line 50 (and the corresponding ft at the end) and then uncomment line 51

Reply:
The data is currently given in the frequency domain. We are doing the calculations in the time domain. So why would I get rid of line 50 here? The data needs to be put into the time domain at this point so that we can carry out the calculations.

Reply:
yes, you would need to ift at some later point in the code.  The first plot is in the frequency domain, so the ift should be moved at least after that.  If you apply the ift to only the temp copy of the data, which I think is possible, then you would not need the ft that happens at the end of the function, currently